main.go

```diff
@@ -26,11 +30,28 @@ func main() {
 		log.Fatal(err)
 	}
 
+	sigs := make(chan os.Signal, 1)
+	abort := make(chan bool, 1)
+	signal.Notify(sigs, syscall.SIGINT, syscall.SIGTERM)
+	go func() {
+		sig := <-sigs
+		io.WriteString(os.Stderr, fmt.Sprintf("Aborting! Caught signal \"%s\"\n", sig))
+		io.WriteString(os.Stderr, "Cleaning up...\n")
+		abort <- true
+		go func() {
```

Comment:
relatedly - is 10 seconds acceptable to your non-Honeycomber opsen persona?

I guess so long as we're outputting somewhere the fact that we're **trying** to exit, 10s isn't too painfully long.